### PR TITLE
What-If Tool progress bar and attribution sorting updates

### PR DESCRIPTION
**Pull request by jameswex**
* Motivation for features / changes

Update loading spinners to be indeterminate progress bars, per UX - in 3 places: datapoint loading, pd plot loading and PD plot sorting.

Add ability to sort attributions by absolute value in sorting dropdown.

* Screenshots of UI changes

button-progress

* Detailed steps to verify changes work correctly (as executed by you)

Ran demos and notebooks with attributions to test new features and visuals.

### CODE REVIEW

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html [hunk 1]
```diff
@@ -7927,13 +7939,16 @@ <h2>Show similarity to selected datapoint</h2>
           return {step: origValue, scalar: origInferenceScore};
         },
 
-        deletePdPlotSpinner: function(featureName) {
+        deletePdPlotProgress: function(featureName) {
           const container = this.featureContainerByName(featureName);
-          deleteElement(container.querySelector('paper-spinner-lite'));
+          const elements = container.querySelectorAll('paper-progress');
```
Comment (tolga-b) [inline tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: why was this change necessary?

Comment (jameswex) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: if you open and close PD plot container fast you can get multiple loading bars created. this just helps ensure if this happens, that when the PD plot is created, all the loading bars go away, not just one of them (same thing was happening with loading circles before this change).

Comment (tolga-b) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: good catch!
